content/user-research/find-user-research-participants/bias.md

```diff
@@ -2,13 +2,13 @@
 layout: text/textblock
 ---
 
-It’s hard to recruit an unbiased sample of user research participants. This is because you’re likely to include some people and exclude others depending on:
+There is a risk that groups of people can be included or excluded, depending on:
 - what the activity is
-- when the sessions are scheduled
-- where you’re doing the research
+- when you schedule the sessions 
+- where you do the research
 - the method you use to recruit your users
 
-The best way to limit this risk is to use a variety of user research activities and recruitment approaches. For example, in Discovery stage don't just rely on interviews. Do some contextual research with people who are using the service and lab research with users of existing services.
+Limit this risk by using a variety of research activities and recruitment approaches. For example, don't just rely on interviews in Discovery stage. Do some contextual research with with users of existing services.
```

Comment:
Delete second 'with' from the line 'Do some contextual research with **with** users of existing services.'